Example/FirestoreSample/FirestoreSample/Views/FavouriteFruitsMappingErrorView.swift

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
+// Copyright 2023 Google LLC
```

Comment:
Don't need to update the copyright of a pre-existing file.

Reply:
Reverted.